src/components/badge/badge.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Badge } from "./badge";
+
+const meta: Meta<typeof Badge> = {
+  component: Badge,
+  argTypes: {
+    variant: {
+      options: ["error", "success", "black", "white"],
```

Comment:
non so se è fattibile però sarebbe figo trovare un modo per estrapolare le keys dalla config di cva e usarle come options

così se in futuro dovessimo aggiornare le varianti, in automatico verrebbero sincronizzate pure qua

```
const badgeStyles = cva(
  "uppercase font-medium text-center inline-flex items-center px-2",
  {
    variants: {
      variant: {
        error: "bg-red",
        success: "bg-green",
        black: "bg-black",
        white: "bg-white",
      },
    },
    ...
```

ps. se è un esplorazione che richiede troppo tempo, per ora lascerei cosi

Reply:
Ottimo punto ma lascerei perdere al momento 😶‍🌫️

Reply:
ci sta